components/angular-tomitribe-select/src/tomitribe-select.ts

```diff
@@ -131,13 +131,27 @@ module tomitribe_select {
                 }
             }
 
-            // multiple selects have no focusser
-            if (uiSelect.multiple) {
-                angular.element(uiSelect.focusInput).on('focus', focusHandler);
-            } else {
-                angular.element(uiSelect.focusser).on('focus', focusHandler);
+            const addListeners = () => {
+              // multiple selects have no focusser
+              if (uiSelect.multiple) {
```

Comment:
I suggest using variable, as multiple/single is creation attribute so should not be changeable in future and two operations is not needed
```let focusEl = angular.element(uiSelect.multiple? uiSelect.focusInput : uiSelect.focusser);```

Reply:
Then both functions can be simplified to a single line